app/db/setup.sh

```diff
@@ -6,6 +6,6 @@ else
     db="realm_registry"
 fi
 
-echo "SELECT 'CREATE DATABASE $db' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '$db')\gexec" | psql -d postgres
+echo "SELECT 'CREATE DATABASE $db' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '$db')\gexec" | psql -U postgres -d postgres
```

Comment:
Was it not working without setting the user explicitly? This might force a password prompt in some setups, although should usually be fine

Reply:
exactly, this seems to work, but I can abastract it away.  I will probably toss a ticket into our backlog for moving this stuff into a make file it's a bit cumbersome as instructions to getting things running, but at least it's documented